src/python/pants/help/help_formatter_test.py

```diff
@@ -2,9 +2,15 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 import unittest
+from dataclasses import replace
+from enum import Enum
 
 from pants.help.help_formatter import HelpFormatter
-from pants.help.help_info_extracter import OptionHelpInfo
+from pants.help.help_info_extracter import HelpInfoExtracter, OptionHelpInfo
+from pants.option.config import Config
+from pants.option.global_options import GlobalOptionsRegistrar
+from pants.option.option_tracker import OptionTracker
+from pants.option.parser import Parser
```

Comment:
We don't want to use all of these imports because then we start coupling this test with the functionality in `pants.option`, i.e. we break the modularity of our tests.

I believe you can simplify lines `test_format_help_choices_enum` to be:

```python
no_default = self.format_help_for_foo(typ=LogLevel)
assert no_default.lstrip() == '--foo=<LogLevel> (one of: [info, debug] default: None)'

default = self.format_help_for_foo(typ=LogLevel, default=LogLevel.INFO)
assert default.lstrip() == '--foo=<LogLevel> (one of: [info, debug] default: debug)'
```

Reply:
Good point, much better to have this test functionality encapsulated.